### ISSUE TITLE
Can only view pull requests.

### ISSUE THREAD

**Issue by rjewell-lower** [MicrosoftDocs/azure-devops-docs#7916]
This workflow doesn't match my experience in Teams.

I can add the extension, but when trying to use it on a channel, I cannot do anything but search for pull requests. When I try to type @Azure repos &lt;anything&gt; it autocompletes to "Pull Request", forcing me to just search for PRs. This prevents me from issuing `@azure repos subscriptions`, to manage notifications in the channel. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 43aa8c60-c25d-535b-2166-f51fa9f171c6
* Version Independent ID: 8890a537-9eda-5606-4be4-580bdc9e8269
* Content: [Azure Repos with Microsoft Teams - Azure Repos](https://docs.microsoft.com/en-us/azure/devops/repos/integrations/repos-teams?view=azure-devops#feedback)
* Content Source: [docs/repos/integrations/repos-teams.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/repos/integrations/repos-teams.md)
* Product: **devops**
* Technology: **devops-code-git**
* GitHub Login: @RGKarthik
* Microsoft Alias: **karrg**

**Comment by WilliamAntonRohm** [comment]
Thanks for your feedback. It looks like you're working through an issue with your scenario or implementation, rather than an issue with the documentation. Here are a couple of options you might consider: 

- [Azure DevOps Support Bot](https://azuredevopsvirtualagent.azurewebsites.net/)  
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops) 

Please [submit your product bug here]( https://developercommunity.visualstudio.com/content/problem/post.html?space=21).

**Comment by PaultheCarter** [comment]
I came here because I experienced the same problem. 

I found the below bug report, which doesn't explicitly state this in the solution, but I think this is happening because we're trying to use the bot in a chat channel instead of a team channel? I can't be sure on that, but if it is the case then I think this page needs to make that an explicit caveat.

I don't yet have access to a team channel so can't test this theory - assuming others will be in the same boat.
​
https://developercommunity.visualstudio.com/content/problem/802070/azure-repos-extension-for-microsoft-teams-will-onl.html
